Source/ACE.Server/Managers/EnchantmentManager.cs

```diff
@@ -35,12 +35,12 @@ public class EnchantmentManager
         /// <summary>
         /// The amount of vitae reduced on player death
         /// </summary>
-        public static readonly float VitaePenalty = 0.05f;
+        public static double VitaePenalty { get { return PropertyManager.GetDouble("vitae_penalty"); } }
```

Comment:
I also wouldn't use a property here.

We used a property here because we had this "magic" value, it needed to go somewhere. However, VitaePenalty really isn't a property of EnchantmentManager, but instead, a property of the shard.